models/tcp_route_test.go

```diff
@@ -3,69 +3,107 @@ package models_test
 import (
 	"encoding/json"
 
-	. "code.cloudfoundry.org/routing-api/models"
+	"code.cloudfoundry.org/routing-api/models"
 
 	. "github.com/onsi/ginkgo/v2"
 	. "github.com/onsi/gomega"
 )
 
+func ptrOfString(s string) *string { return &s }
```

Comment:
This function along with the refactor with the `JustBeforeEach` is much nicer.
I would rename this function to `pointerToString` or `ptrToString` to make it even more clear.

Reply:
Fixed